aws/basic-eks-cluster/blueprint.yaml

```diff
@@ -123,6 +124,29 @@ spec:
     - p3.8xlarge
     - p3.16xlarge
 
+  - name: AddIAMUser
+    description: "Do you want to add extra user to have an access to EKS via kubectl"
+    type: Confirm
+    default: true
+
+  - name: EKSUserARN
+    description: "Enter existent IAM user ARN to be used to access EKS cluster via kubectl"
+    type: Input
+    default: "arn:aws:iam::100000000000:user/username"
+    pattern: ^arn:aws:iam::\d{12}:(?:root|user\/[A-Za-z0-9]+)$
```

Comment:
This pattern doesn't support dashes in "username" part. 
Value arn:aws:iam::932770550094:user/test-eks was not validated by the patter though it's a valid AWS IAM ARN.

Reply:
Updated regex pattern